src/core/particle_data.cpp

```diff
@@ -2304,8 +2295,3 @@ bool particle_exists(int part) {
         return true;
    return false;
 }
-
-#ifndef MASS
-constexpr double ParticleProperties::mass;
```

Comment:
I included the declaration again at https://github.com/KaiSzuttor/espresso/blob/4c15c144bc763e603cf910b1a50a36fcef2ae3e4/src/core/particle_data.cpp#L705-L707